src/main/resources/assets/stylesheets/iva-msva-30.hbs

```diff
@@ -0,0 +1,275 @@
+/* Styling shared by IVA30 and MSVA30 */
```

Comment:
Why is it sharing staying when the iva 30 has a separate file for styling?

Reply:
can we update the comment to reflect this please?